integr8ly/installation
# Intly 218 amq streams

## Additional Information
Jira: https://issues.jboss.org/browse/INTLY-1553

This PR adds:
- The AMQ Streams operator (https://github.com/integr8ly/installation/commit/7ee0208484858e9a17a2277c8d74baa0eab7e565)
- The above tasks created namespaces so there is a commit to consolidate namespace creation (https://github.com/integr8ly/installation/commit/82427e476b63168ababb6282af7b0e8bd1ecc9c6)
## Verification Steps

1. Run this branch. The new playbooks are not run by default so you need to pass a param.
```
 -e amq_streams=true
```

3. There is currently no status set on the Kafka resource used to create the cluster but the AMQ Streams namespace should look like:
![amqstreams](https://user-images.githubusercontent.com/22113654/56977670-9eb85a80-6b6d-11e9-8add-781e15e520c9.png)

4. uninstall passing these params:
```
 -e amq_streams=true
```
